### DISCUSSION TITLE
Unknown GUI toolkit

### DISCUSSION THREAD

**Discussion by Trishankgoswami** [eclipse/windowbuilder#736] [General]
Hi 

there an error i am facing while opening the Design page i have tried my best to resolve it by doing google & you tube. if some one t help me to resolve it.  
as i am trying to make a application
![Screenshot 2024-03-26 220153](https://github.com/eclipse-windowbuilder/windowbuilder/assets/165075112/260ad55a-c350-4729-86b2-2f337be98627)
 for windows.

**Comment by ptziegler** [comment]
Can you post the source code of Skkip.java? As the error message says, WindowBuilder doesn't understand the way the JFrame instance is created in this class.

**Reply by Trishankgoswami** [reply]
ok please see.
![Screenshot 2024-03-27 124952](https://github.com/eclipse-windowbuilder/windowbuilder/assets/165075112/07b78078-e810-4f68-92ec-629ef0b7f1d0)

**Reply by ptziegler** [reply]
Thanks!
Then let's start with the obvious: With so many compilation errors, there is no way this could every be read by the WindowBuilder editor.

I see the `module-info.java` in the second tab, so I assume you're using [modules](https://www.oracle.com/corporate/features/understanding-java-9-modules.html). If you don't want to use it or are unsure how exactly they work, I would suggest to simply delete this file. Alternatively, you need to require at least `java.desktop`.

There are also other imports missing, such as `java.lang.Runnable`. Those need to be resolved as well, before the class can compile.

In the initial screenshot, I also see error markers in the Java project. That looks a lot like something is broken in the Java project itself.

Can you also post the errors that are shown in the `Problems` view?

**Comment by Trishankgoswami** [comment]
ok 
![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/165075112/df8c5d90-041d-486b-aa29-c7c7ef24fea2)
![Uploading Screenshot 2024-03-27 124952.png…]()

**Comment by miguelramirezc** [comment]
Thanks, I have the same problem, and just delete the the module-info.java, then window builder works
